vendor/bat-native-ads/src/bat/ads/internal/static_values.h

```diff
@@ -29,6 +29,9 @@ const int kWinningCategoryCountForServingAds = 3;
 // confirmation types
 const uint64_t kMaximumEntriesInAdsShownHistory = 7 * (20 * 4);
 
+// TODO(Moritz Haller): Adjust history per segment cap if too generous
```

Comment:
Before merging can we make sure we are happy with 100, and then remove the comment?